### PR TITLE
Theme and Overhaul the CI incident page.

### PR DESCRIPTION
**Pull request by nedSaf**
#168

### CODE REVIEW

**Comment by nedSaf** [comment]
@amitaibu What do you think about the information labels ?
localhost_9000__code_348fcfe6a8914e8f9f62__ci-incidents_27

**Comment by nedSaf** [comment]
Incident info examples:
localhost_9000__code_348fcfe6a8914e8f9f62__ci-incidents_28

localhost_9000__code_348fcfe6a8914e8f9f62__ci-incidents_27

File: client/app/views/dashboard/ci_incident.html [hunk 1]
```diff
@@ -1,12 +1,60 @@
...
+            Terminated: {{ (incident.updated*1000) | date : 'MMM d, y h:mm a' }}
+          </span>
+          <span class="label label-sm label-queue incident-info">
+            Downtime: {{ downTime }}
+          </span>
+        </div>
+      </div>
+      <div class="portlet-body">
+        <ul class="info incident-builds">
+          <li>Fail build: <a ui-sref="dashboard.ciBuildItem({ ciBuildItemId: incident.failing_build.id })">Build #{{ incident.failing_build.id }}</a></li>
```
Comment (amitaibu) [inline client/app/views/dashboard/ci_incident.html]: 1st failing build

File: client/app/views/dashboard/ci_incident.html [hunk 2]
```diff
@@ -1,12 +1,60 @@
...
+          <li ng-if="incident.fixed_build">Fix build: <a ui-sref="dashboard.ciBuildItem({ ciBuildItemId: incident.fixed_build.id })">Build #{{ incident.fixed_build.id }}</a></li>
```
Comment (amitaibu) [inline client/app/views/dashboard/ci_incident.html]: Fixed build

**Comment by nedSaf** [comment]
Changed build names:
localhost_9000__code_348fcfe6a8914e8f9f62__ci-incidents_30

**Comment by nedSaf** [comment]
@amitaibu Please review #184 first. (log style is on that branch).

**Comment by amitaibu** [comment]
failing/ fixed build ID should start on the same horizontal line.

**Comment by nedSaf** [comment]
default

Mobile: 
localhost_9000__code_c8ca347dbea72fdfb2b5__ci-incidents_30

**Comment by amitaibu** [comment]
Needs a re-roll

**Comment by nedSaf** [comment]
Incident page with log:
localhost_9000__code_327ea5776341ee13e1fc__ci-incidents_27

**Comment by nedSaf** [comment]
@amitaibu Ready.

File: client/app/scripts/controllers/ci_incident.js [hunk 3]
```diff
@@ -8,7 +8,32 @@
...
+  .controller('CiIncidentCtrl', function ($scope, incident, Builds) {
 
     $scope.incident = incident[0];
+
+    Builds.get($scope.incident.ci_build, 'ci_build').then(function(data) {
+      $scope.build = data[0];
+    });
+
+    var incidentStart = $scope.incident.created;
+    var incidentEnd = $scope.incident.created != $scope.incident.updated ? $scope.incident.updated : (new Date().getTime())/1000;
```
Comment (amitaibu) [inline client/app/scripts/controllers/ci_incident.js:20]: incidentEnd should be determined by checking if there's a reference to `fixed_build`

File: client/app/views/dashboard/ci_incident.html [hunk 4]
```diff
@@ -1,12 +1,60 @@
...
+    <div class="portlet">
+      <div class="portlet-title">
+        <div class="caption">
+          <span class="caption-subject">{{ incident.label }}</span>
+        </div>
+        <div class="actions">
+          <span class="label label-sm label-error incident-info">
+            Started: {{ (incident.created*1000) | date : 'MMM d, y h:mm a' }}
+          </span>
+          <span ng-if="incident.created != incident.updated" class="label label-sm label-done incident-info">
```
Comment (amitaibu) [inline client/app/views/dashboard/ci_incident.html:36]: use `fixed_build` info, not `updated`

**Comment by amitaibu** [comment]
Opened follow up in #194

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**